report-pull-request-results.py

```diff
@@ -24,7 +24,7 @@
 parser = OptionParser(usage="usage: %prog ACTION [options] \n ACTION = PARSE_UNIT_TESTS_FAIL | PARSE_BUILD_FAIL "
                             "| PARSE_MATRIX_FAIL | COMPARISON_READY | GET_BASE_MESSAGE | PARSE_EXTERNAL_BUILD_FAIL "
                             "| PARSE_ADDON_FAIL | PARSE_CRAB_FAIL | PARSE_CLANG_BUILD_FAIL | MATERIAL_BUDGET "
-                            "| PYTHON3_FAIL | MERGE_COMMITS")
+                            "| PYTHON3_FAIL | PARSE_GPU_FAIL | MERGE_COMMITS")
```

Comment:
also rename `PARSE_GPU_FAIL` to `PARSE_GPU_UNIT_TESTS_FAIL` and use it https://github.com/cms-sw/cms-bot/pull/1973/files#diff-20b0d9147b748bf949bfbb0e02903ecceb5d3d86bcbe1cca396c03e27eb84f5eR37 ( i.e. replace  `PARSE_UNIT_TESTS_FAIL` with `PARSE_GPU_UNIT_TESTS_FAIL`